Tests/BackendIntegrationTests/BaseBackendIntegrationTests.swift

```diff
@@ -86,6 +88,14 @@ class BaseBackendIntegrationTests: XCTestCase {
         self.mainThreadMonitor = nil
     }
 
+    /// Simulates closing the app and re-opening with a fresh instance of `Purchases`.
+    final func resetPurchases() async {
```

Comment:
Nit: naming could be a bit better, as it could be semantically confused with resetting "purchases of entitlements" rather than the "Purchases Singleton"

Reply:
I agree 👍🏻 I think I'll rename to `resetSingleton`